src/graphql/mutation/createInsertMutationField.js

```diff
@@ -60,12 +63,45 @@ const createPayloadType = table =>
       [table.getFieldName()]: {
         type: createTableType(table),
         description: `The inserted ${table.getMarkdownTypeName()}.`,
-        resolve: source => source.output,
+        resolve: ({ output }) => output,
+      },
+
+      [`${table.getFieldName()}Edge`]: {
+        type: createTableEdgeType(table),
+        args: {
+          orderBy: {
+            type: createTableOrderingEnum(table),
+          },
+        },
+        description: 'An edge to be inserted in a connection with help of the containing cursor.',
+        resolve: resolveNewEdge(table),
       },
+
       ...getPayloadFields(table.schema),
     },
   })
 
+const resolveNewEdge = table => ({ output }, { orderBy }) => {
+  // See http://stackoverflow.com/a/32615799/1140494
+  let cursor = null
+
+  // if the edge field was handed an orderBy argument we use
+  // this as the cursor
+  if (orderBy) {
+    cursor = output[orderBy]
+  }
+  // if orderBy is not present use the primary keys of the table
+  else {
+    const primaryKeys = table.getPrimaryKeys()
+    cursor = primaryKeys.map(column => output[column.name])
```

Comment:
Instead of doing this, could we use a default value for the argument same as what we do [here](https://github.com/calebmer/postgraphql/blob/master/src/graphql/createConnectionArgs.js#L16)? This means calling `fooEdge` without arguments would result in the exact same cursor you would get from the connection without an `orderBy` argument.